models/lung_nodule_ct_detection/configs/inference.json

```diff
@@ -71,16 +73,18 @@
         "_target_": "Compose",
         "transforms": [
             {
-                "_target_": "DeleteItemsd",
-                "keys": [
-                    "box",
-                    "label"
-                ]
+                "_target_": "LoadImaged",
+                "keys": "image",
+                "meta_key_postfix": "meta_dict",
+                "_disabled_": "@whether_raw_luna16"
```

Comment:
Hi @Can-Zhao , thanks for the updates. Here I think we may not need two `LoadImaged `, and you can use:
```
            {
                "_target_": "LoadImaged",
                "keys": "image",
                "meta_key_postfix": "meta_dict",
                "reader": "$'itkreader' if @whether_raw_luna16 else 'none'",
                "affine_lps_to_ras": "@whether_raw_luna16",
            },
```
In addition, according to the readme file, should the `affine_lps_to_ras` of `AffineBoxToWorldCoordinated` be consistent with this transform?

Reply:
I'm afraid they cannot be combined.
1) the resampled images are nifti. nifti reader does not take `affine_lps_to_ras` as an argument, it will give an error.
2) that's why I put two `"LoadImaged"` here.

Reply:
Thanks @Can-Zhao , I double checked the settings with nii format data and you are right.
Could you please also update the changelog and version in metadata as I mentioned in a previous comment? It is a necessary step before merging a PR, thanks again!

Reply:
Thank you! Just changed them!